lms/templates/courseware/info.html

```diff
@@ -103,7 +103,6 @@ <h3 class="hd hd-3 handouts-header">${_("Course Tools")}</h3>
           % if SelfPacedConfiguration.current().enable_course_home_improvements:
             ${HTML(dates_fragment.body_html())}
           % endif
-          <h3 class="hd hd-3  handouts-header">${_(course.info_sidebar_name)}</h3>
```

Comment:
I think this is pending the decision on course_info itself.

Reply:
It's not, the question was about the Studio page, which will probably still be needed. We should remove the LMS page, because this is what we've effectively removed in `lms/djangoapps/courseware/views/views.py`.